README.md

```diff
@@ -1 +1,5 @@
-# Just another github repo
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
I suggest you can use the README.md 📝  document to describe your project to make it understandable for other developers. consider using the [template](https://github.com/microverseinc/readme-template).